src/components/BusinessCard/BusinessCard.jsx

```diff
@@ -70,79 +71,55 @@ const BusinessCard = ({
   address,
   distance,
   filters,
-}) => {
-  const chipRenderer = () => {
-    const filtersToRender = filters.slice(0, 4);
-    const chips = filtersToRender.map((filter) => (
-      <Chip
-        color="secondary"
-        label={filter.name}
-        className={classes.filter}
-        key={filter.name.replace(/\w/, '')}
-      />
-    ));
-    return chips;
-  };
-  return (
-    <Card className={classes.root}>
-      <CardHeader
-        avatar={(
-          <div color="secondary">
-            <StarIcon color="secondary" fontSize="large" />
-            <Typography
-              variant="body2"
-              color="secondary"
-              className={classes.rating}
-            >
-              {averageRating}
-            </Typography>
-          </div>
-        )}
-        action={<BookmarkBorderIcon fontSize="large" color="secondary" />}
-        title={<Typography variant="h6">{name}</Typography>}
-        subheader={<Typography variant="body2">{category}</Typography>}
-      />
-      <CardMedia
-        image={imageUrl}
-        // todo make this responsive
-        className={classes.cardMedia}
-      />
-      <CardContent>
-        <div className={classes.location}>
-          <div className={classes.address}>
-            <Typography variant="body1">{address}</Typography>
-          </div>
-          <div className={classes.distance}>
-            <Typography variant="body1">{distance}</Typography>
-          </div>
+}) => (
+  <Card className={classes.root}>
+    <CardHeader
+      avatar={(
+        <div color="secondary">
+          <StarIcon color="secondary" fontSize="large" />
+          <Typography
+            variant="body2"
+            color="secondary"
+            className={classes.rating}
+          >
+            {averageRating}
+          </Typography>
         </div>
-        <Divider />
-        <div className={classes.chipWrapper}>
-          {chipRenderer()}
-          {filters.length > 5 ? (
-            <Chip
-              color="secondary"
-              variant="outlined"
-              label={`+${filters.length - 5} More`}
-              className={classes.filter}
-            />
-          ) : (
-            ''
-          )}
+      )}
+      action={<BookmarkBorderIcon fontSize="large" color="secondary" />}
+      title={<Typography variant="h6">{name}</Typography>}
+      subheader={<Typography variant="body2">{category}</Typography>}
+    />
+    <CardMedia
+      image={imageUrl}
+      // todo make this responsive
```

Comment:
I'm guessing like this: https://material-ui.com/components/cards/#RecipeReviewCard.js
It's not part of this ticket though is it?